### PR TITLE
update sles15 sap image family

### PR DESCRIPTION
**Pull request by braydonk**
## Description
Adding the new image family `sles-15-sp6-sap` to our tests.

## Related issue
[b/349706126](http://b/349706126)

## How has this been tested?
Integration tests

## Checklist:
- Unit tests
  - [x] Unit tests do not apply.
  - [ ] Unit tests have been added/modified and passed for this PR.
- Integration tests
  - [ ] Integration tests do not apply.
  - [x] Integration tests have been added/modified and passed for this PR.
- Documentation
  - [x] This PR introduces no user visible changes.
  - [ ] This PR introduces user visible changes and the corresponding documentation change has been made.
- Minor version bump
  - [x] This PR introduces no new features.
  - [ ] This PR introduces new features, and there is a separate PR to bump the [minor version](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/VERSION) since the last [release](https://github.com/GoogleCloudPlatform/ops-agent/releases) already.
  - [ ] This PR bumps the version.

### CODE REVIEW

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

File: integration_test/gce/gce_testing.go [hunk 1]
```diff
@@ -989,9 +989,8 @@ func prepareSLES(ctx context.Context, logger *log.Logger, vm *VM) error {
 		// --gpg-auto-import-keys is included to fix a rare flake where (due to
 		// a policy being installed already) there is a new key that needs to
 		// be imported.
-		// timezone-java was selected arbitrarily as a package that:
-		// a) can be installed from the default repos, and
-		// b) isn't installed already.
+		// To fix this, we force install a package. coreutils is arbitrarily
+		// chosen as it's all but guaranteed to be present.
 		_, zypperErr := RunRemotely(ctx, logger, vm, "sudo zypper --non-interactive --gpg-auto-import-keys refresh && sudo zypper --non-interactive install timezone-java")
```
Comment (igorpeshansky) [inline integration_test/gce/gce_testing.go:994]: I suspect you meant to actually change this to `coreutils`, i.e.:
```suggestion
		_, zypperErr := RunRemotely(ctx, logger, vm, "sudo zypper --non-interactive --gpg-auto-import-keys refresh && sudo zypper --non-interactive install --force coreutils")
```

Comment (braydonk) [reply integration_test/gce/gce_testing.go]: Yep, I did mean to change it lol. copy paste mishap

Done in [1f79427](https://github.com/GoogleCloudPlatform/ops-agent/pull/1743/commits/1f79427556d6133cba3ec79ff915e404414bd62b)

File: integration_test/third_party_apps_test/applications/postgresql/sles/install [hunk 2]
```diff
@@ -14,6 +14,14 @@ if [[ "${SUSE_VERSION}" == 12 ]]; then
   sudo zypper --no-gpg-checks in -y postgresql14-server=14.1-3.3.1 postgresql14-contrib=14.1-3.3.1 postgresql14-libs=14.1-1PGDG.sles12 postgresql14=14.1-3.3.1
 elif [[ "${SUSE_VERSION}" == 15 ]]; then
   sudo zypper addrepo https://download.postgresql.org/pub/repos/zypp/repo/pgdg-sles-15-pg14.repo
+
+  # VERSION_ID is taken from /etc/os-release, and is of the form <SLES version number>.<Service Pack number>.
+  if [[ "$VERSION_ID" == "15.6" ]]; then
+    # TODO(b/350987619): There is currently no "15.6" repo for PostgreSQL 14. Since the automatic
```
Comment (dehaansa) [inline integration_test/third_party_apps_test/applications/postgresql/sles/install:20]: This comment feels like it is only half-written?

**Review by dehaansa — APPROVED** [review]

File: integration_test/third_party_apps_test/applications/postgresql/sles/install (hunk 2 above)
Comment (braydonk) [reply integration_test/third_party_apps_test/applications/postgresql/sles/install:20]: Oops! Finished in [b07ac24](https://github.com/GoogleCloudPlatform/ops-agent/pull/1743/commits/b07ac2428a32dcc91580d05956e2d51d4b08a868)

**Comment by braydonk** [comment]
The remaining CI failures were due to the test-vendors repo mishap in our test infrastructure and EOL distros that are in the process of being removed. Most recent commit was just a comment that won't change the previous CI outcome.

Based on this, merging the PR now.

**Pull request merged**